torch/_dynamo/variables/functions.py

```diff
@@ -1259,9 +1258,9 @@ def call_function(
             ]
             block_dims = [
                 kwargs["block_dim1"] if "block_dim1" in kwargs else args[3],
-                kwargs["block_dim2"] if "block_dim2" in kwargs else args[4],
+                kwargs["block_dim0"] if "block_dim0" in kwargs else args[4],
             ]
-        element_size = kwargs["ptr"] if "ptr" in kwargs else args[-1]
+        element_size = kwargs["element_size"] if "element_size" in kwargs else args[-1]
```

Comment:
```suggestion
        element_size = kwargs.get("element_size", args[-1])
````

Reply:
And for all the other kwargs accesses with the similar type, should avoid accidental typos of the key